modules/account-lib/src/coin/stx/utils.ts

```diff
@@ -0,0 +1,156 @@
+import BigNumber from 'bignumber.js';
+import { bufferToHex, stripHexPrefix } from 'ethereumjs-util';
+import {
+  AddressHashMode,
+  StacksTransaction,
+  TransactionVersion,
+  addressFromVersionHash,
+  addressHashModeToVersion,
+  addressToString,
+  validateStacksAddress,
+} from '@stacks/transactions';
+
+/**
+ * Encodes a buffer as a "0x" prefixed lower-case hex string.
+ *
+ * @param {Buffer} buff - a buffer with a hexadecimal string
+ * @returns {string} - the hexadecimal string prefixed with "0x"
+ */
+export function bufferToHexPrefixString(buff: Buffer): string {
+  return bufferToHex(buff);
+}
+
+/**
+ * Remove the "0x" prefix from the given string, if present.
+ *
+ * @param {string} hex - a hexadecimal string
+ * @returns {string} - the hexadecimal string without a leading "0x"
+ */
+export function removeHexPrefix(hex: string): string {
+  return stripHexPrefix(hex);
+}
+
+/**
+ * @param publicKeyHash
+ * @param hashMode
+ * @param transactionVersion
+ */
+function getAddressFromPublicKeyHash(
```

Comment:
How do we get an address from just one public key? Is this capable of creating a multisig address? Are we doing single sig for STX? Feels like I'm missing something here.

Reply:
This code is part of the transactions PR.  I can remove it for now if needed.